srcpkgs/nfdump/files/nfcapd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec nfcapd -u _nfcapd -g _nfcapd -l /var/log/nfdump $OPTS
```

Comment:
Could `/var/log/nfdump` be changed to ${LOGDIR:=/var/log/nfdump} to make it easier to override the destination?

Reply:
Should probably just be `${OPTS:--l /var/log/nfdump}`.